vitality/templates/workout/workout.html

```diff
@@ -1,6 +1,7 @@
 {% extends 'base.html' %}
 {% block content %}
+<h1>Workout</h1>
 <div class="row"> 
-    
+    <p> {{ workoutInfo }} </p>
```

Comment:
You were close on this one. 

- Firstly the `<h1>` tag should be inside the `<div class="row">` tag. 
- workoutInfo should be a Workout class so you will need to specify all attributes explicily. 
  - This means it will be different `<p>` tags with `Name: {{ workout.name }}`, `About: {{ workout.about }}`, etc... 
  - You can ignore the `_id` attribute. 
- Also surround all the `<p>` tags with one `<div class = "col-sm-12">` tag for responsiveness.